README.md

```diff
@@ -43,6 +43,7 @@ Or install it yourself as:
   exchange_durable true # optionally set exchange durability - default is true.
   payload_only false # optional - default is false. if true, only the payload will be sent. if false, data format is { "key" => tag, "timestamp" => time, "payload" => record }.
   content_type application/octet-stream # optional - default is application/octet-stream. some amqp consumers will expect application/json.
+  priority 0 # the priority for the message
```

Comment:
Could you add the comment of the minimum `bunny` version for this option and minimum `rabbitmq`?